packages/osd-babel-preset/common_preset.js

```diff
@@ -29,30 +29,23 @@
  */
 
 const plugins = [
+  '@babel/plugin-transform-class-properties',
+  '@babel/plugin-transform-private-methods',
   require.resolve('babel-plugin-add-module-exports'),
 
-  // The class properties proposal was merged with the private fields proposal
-  // into the "class fields" proposal which is stage 3.
-  // See https://github.com/babel/proposals/issues/12 for progress
-  require.resolve('@babel/plugin-proposal-class-properties'),
-
   // Optional Chaining proposal is stage 4 (https://github.com/tc39/proposal-optional-chaining)
   // Need this since we are using TypeScript 3.7+
```

Comment:
These proposal comments now all seem outdated (particularly because they reference previous version of TypeScript). But I'm not sure I know enough about the relationship between ES proposals, Typescript, and Babel to know how to investigate whether we still need these plugins at all or to update the comments to explain why we still need them. Any help is appreciated!